### ISSUE TITLE
Suggestion: Subdirectory structure for albums example based on user settings

### ISSUE THREAD

**Issue by LegoChicken** [jcgruenhage/PlayMusicExporter#26]
In Settings, Export, Subdirectory structure for albums there is "Example: Beatles/Help!/13. Yesterday.mp3". Could you make this example based on the actual "Subdirectory structure for albums" setting. This would make it easy to check that you had got the expected setting. I made mistake the first time I changed this setting and couldn't see the "wrong" naming until I had done the export. Obviously the same would also apply to "Subdirectory structure for playlists".

**Comment by jcgruenhage** [comment]
You mean, parsing the structure and printing out the result for the example song? Sounds like a very doable (and nice) feature, thanks for the suggestion.

**Comment by LegoChicken** [comment]
Yes, so for example I use:
{album-artist}/{album}/{disc=CD $}/{album-artist} - {no=$$} - {title}.mp3
So this should show:
Example: Beatles/CD 1/Help!/Beatles - Help! - 01 - Yesterday.mp3
Is possible to detect if there is more than 1 CD for album. If so, the disc part could be omitted.
A more complex addition would be to also have an option for "various artists" naming scheme. This would have a different layout. For example, I use "{album}/{album} - {no=$$} - {artist} - {title}.mp3".

**Comment by jcgruenhage** [comment]
I've already thought about the show disc only, if there are multiple discs thing, I'll most certainly do that some time in the future, I don't think this is really high priority though.

**Comment by LegoChicken** [comment]
I agree that it isn't high priority. Its easy to move the files between directories afterwards. Not as easy to rename them.
